### PR TITLE
Update Identity Administration and Provisioning and Managing Claims sections

### PR DESCRIPTION
**Pull request by YvonneW**
The following Workflow Management pages were updated:

- `docs/learn/adding-a-new-workflow-definition.md`
-  `docs/learn/engaging-a-workflow-in-an-operation.md`
- `docs/learn/managing-human-tasks.md`
- `docs/learn/monitoring-workflow-requests.md`
- `docs/learn/workflow-management.md`

Following files were removed and `mkdocs.yml` was updated as they were pointing to BPS docs :
- `docs/learn/adding-a-workflow-engine.md`
- `docs/learn/configuring-the-bpm-profile-of-wso2-ei-as-a-workflow-engine.md`

Updated the folling doc pages:
- `docs/learn/associating-user-accounts.md`
- `docs/learn/creating-and-managing-tenants.md`
- `docs/learn/customizing-automated-emails.md`
- `docs/learn/enabling-notifications-for-user-operations.md`
- `docs/learn/managing-consent-purposes.md`
- `docs/learn/managing-user-attributes.md`
- `docs/learn/using-email-address-as-the-username.md`
- `docs/learn/using-the-end-user-dashboard.md`

Updated `docs/develop/managing-claims-with-apis.md`

### CODE REVIEW

File: en/docs/learn/associating-user-accounts.md [hunk 1]
```diff
@@ -15,20 +15,19 @@ have logged on.
     ``` toml
```
Comment (gomathyk) [inline en/docs/learn/associating-user-accounts.md]: fix indentation

File: en/docs/learn/associating-user-accounts.md [hunk 2]
```diff
@@ -15,20 +15,19 @@ have logged on.
...
-    
-    !!! tip
-        The recommended approach is to have the
-        `         EnableFederatedUserAssociation        ` parameter set to
-        `         false        ` so that manual federated user association is
-        not allowed by default via the dashboard.
+    ```    
+
+    The **recommended approach** is to have the
+    `         EnableFederatedUserAssociation        ` parameter set to
```
Comment (gomathyk) [inline en/docs/learn/associating-user-accounts.md]: Is this a toml property? if not, please replace with the toml equivalent.

File: en/docs/learn/customizing-automated-emails.md [hunk 3]
```diff
@@ -140,19 +100,11 @@ template.
         You can also specify any custom claim URI created in the WSO2 claim
         dialect:
     
-        -   If it an identity claim, specify the claim as seen in the
-            example below.
-    
-            The claim <http://wso2.org/claims/identity/homeAddress> should
-            be specified as
-            `               {{user.claim.identity.homeAddress}}              `
+        -   If it an identity claim, specify the claim accordingly. For example, the claim `<http://wso2.org/claims/identity/homeAddress>` should be specified as
```
Comment (gomathyk) [inline en/docs/learn/customizing-automated-emails.md]: If it **is** an identity claim,

File: en/docs/learn/engaging-a-workflow-in-an-operation.md [hunk 4]
```diff
@@ -1,70 +1,41 @@
...
-        definition](../../learn/adding-a-new-workflow-definition)
-        .
+    -   **Operation name**: This is the operation to be selected.
 
-    5.  The Application condition:
+    -   **Workflow**: This is the workflow that you need to engage for the event. You may either use an existing workflow definition or [create a workflow definition](../../learn/adding-a-new-workflow-definition).
 
-    !!! info 
-        There are three options which we can use to filter requests which
+    -   **The Application conditiona**: There are three options which we can use to filter requests which
```
Comment (gomathyk) [inline en/docs/learn/engaging-a-workflow-in-an-operation.md]: Remove a in condition.

File: en/docs/learn/managing-human-tasks.md [hunk 5]
```diff
@@ -18,51 +15,30 @@ IS as well. To test the workflow, follow the steps below.
...
-    approval flow. For example, a workflow with two step approval where in
-    the first step the role to authorize the request is the M anager and in
-    the second step the role to authorize the request is the S eniorManager
-    . In this case, when a user is added to the system, only users with M
-    anager role can initially see the pending approvals. After the users
-    with this role approves the pending tasks, then the users with S
-    eniorManager role can see the pending tasks. After both the users
-    approve, the task is completed.
+    Instead of a one-step approval, there can be multiple steps to the approval flow. For example, a workflow with two step approval where in the first step the role to authorize the request is the M anager and in the second step the role to authorize the request is the S eniorManager. In this case, when a user is added to the system, only users with Manager role can initially see the pending approvals. After the users with this role approves the pending tasks, then the users with SeniorManager role can see the pending tasks. After both the users approve, the task is completed.
```
Comment (gomathyk) [inline en/docs/learn/managing-human-tasks.md]: Remove the extra space in Manager and Senior Manager.

File: en/docs/learn/associating-user-accounts.md (hunk 1 above)
Comment (YvonneW) [reply en/docs/learn/associating-user-accounts.md]: Fixed

**Pull request merged**